src/main/java/net/dv8tion/jda/core/entities/Emote.java

```diff
@@ -135,6 +135,15 @@
      */
     EmoteManagerUpdatable getManagerUpdatable();
 
+    /**
+     * Whether or not this Emote is animated or not.
+     *
+     * <p>Animated Emotes are available only to Discord Nitro users.
```

Comment:
Not sure if this is relevant info.

Reply:
tbh, I wasn't quite sure what or how to document this.
I figured it might be useful for people using `AccountType.CLIENT` to know, as they cannot use the emotes without having Nitro.
If anything, I should add that it's also available to all `AccountType.BOT` clients, but I can understand if someone sees it as irrelevant.

Reply:
What happens if you try to use an animated emote with a non-nitro client account?

Reply:
They will not display as far as I can tell.
the new format is not supported for non supported accounts

Reply:
Then it should be checked in Member#canInteract(Emote) etc.

Reply:
I was thinking the same thing!